### ISSUE TITLE
Baubles and CAR item dupes

### ISSUE THREAD

**Issue by TheKrister2** [iChun/Sync#208]
When transferring to a new shell, all baubles, and cosmetic armor worn using the mod cosmetic armor reworked. Soul bound items like my sword were also duplicated. Have checked to make sure, but Levels 2 (Now renamed to Weapon Levels) is not present. Currently using Sync 7.0.2.

**Comment by daAresinger** [comment]
I won't make a separate issue, unless told to do otherwise:
For me the entire Inventory is duped every time I switch Shells.
When I then switch I take my current inventory with me without deleting it on the shell I am leaving.

I do not have Weapon Levels installed, nor Cosmetic Armor. However I do have Baubles and Tinkers Construct. (aswell as countless other mods, but these two have come up in this problem before).

This is a video threw together before I knew this was a known issue:
https://youtu.be/eTm3gCt5tXw

If you are interested I can export the Minecraft instance and upload it somewhere (MultiMC Instance)

**Comment by ichttt** [comment]
Both of you please try https://github.com/iChun/Sync/files/1890896/Sync-1.12.2-7.0.3BETA.zip and see if it fixes the issue

**Comment by daAresinger** [comment]
It fixed the duping. My inventory (Including Baubles) is now persistent between all shells. Is this intended?

**Comment by ichttt** [comment]
Nope, not intended. Please provide a modlist so I can track the issue.

**Comment by ichttt** [comment]
Here is another testing build with a config option to disable the new item sync mechanism if it breaks with some of your mods. Search for "Try tranfering persistent items" and set it to no in the config gui (or 0 if you edit the file), and please report back if everything is as you exspected. Just unzip the zip file and replace the original sync jar with this one.
[Sync-1.12.2-7.0.3BETA2.zip](https://github.com/iChun/Sync/files/2028086/Sync-1.12.2-7.0.3BETA2.zip)

**Comment by TheKrister2** [comment]
Have tested it with "Try transferring persistent items" on and off, neither fixes the issue for me. Baubles and Cosmetic Armor Reworked items stay within their slots, though it seems that when "Try transferring persistent items" was set to 'yes' the items did not duplicate but rather moved over to a new shell instead of staying with the shell they were in originally.

[Mod list.](https://pastebin.com/G9fuKcbK)

**Comment by daAresinger** [comment]
sorry, I couldn't check any sooner. It works fine for my normal inventory when I turn the option off. But baubles is still persistent.
